test/e2e/util.go

```diff
@@ -708,82 +744,91 @@ func RunRC(c *client.Client, name string, ns, image string, replicas int) error
 	By(fmt.Sprintf("%v Waiting for all %d replicas to be running with a max container failures of %d", time.Now(), replicas, maxContainerFailures))
 	same = 0
 	last = 0
-	failCount = 10
+	failCount = int(100 / interval)
```

Comment:
Same reasoning as the other failCount.  Original code 10 failures, 10 second check interval